nextcord/threads.py

```diff
@@ -29,6 +29,7 @@
 import time
 from typing import TYPE_CHECKING, Callable, Dict, Iterable, List, Optional, Union
 
+from . import channel
```

Comment:
Why is the channel module imported instead of what's needed from the channel module?

Reply:
Import cycles.

Reply:
Can you explain how directly importing the module circumvents import cycles? I don't quite understand.

Reply:
Importing the module itself means the whole module does not need to be evaluated to find out what is importable.

Reply:
Ah okay.